Source/Shaders/SSR.shader

```diff
@@ -148,7 +149,9 @@ float4 PS_RayTracePass(Quad_VS2PS input) : SV_Target0
         float3 hitPosition = sdfHit.GetHitPosition(sdfTrace);
         float surfaceThreshold = GetGlobalSurfaceAtlasThreshold(GlobalSDF, sdfHit);
         float4 surfaceAtlas = SampleGlobalSurfaceAtlas(GlobalSurfaceAtlas, GlobalSurfaceAtlasChunks, RWGlobalSurfaceAtlasCulledObjects, GlobalSurfaceAtlasObjects, GlobalSurfaceAtlasDepth, GlobalSurfaceAtlasTex, hitPosition, -reflectWS, surfaceThreshold);
-        result = lerp(surfaceAtlas, float4(result.rgb, 1), result.a);
+        // Now the sdf reflection part is significantly darker than the screen space part
+		// TODO: Maybe multiply surfaceAtlas by a constant to make it brighter, adding 2* looks fine
```

Comment:
Have you tested this in some Flax Samples? Maybe we need to include some PreExposure or Eye Adaptation to balance the intensity of the relfections?

Reply:
Yeah, a clear boundary between the screen space part and the SDF part could be seen. I found that the problem is more series when it comes to forward pass. But I suppose it's the SSR part being too bright for in the forward pass.